code/script.js

```diff
@@ -221,105 +269,174 @@ const generateBoard = () => {
     `
   })
 }
-
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  console.log(secret);
 }
 
-// This function to start (and restart) the game
+/// This function to start (and restart) the game
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
+  board.style.display = 'flex'
+  winOrLose.style.display = 'none'
   // What else should happen when we start the game?
+  //Game board should be rendered on the screen
+  generateBoard();
+  setSecret();
+  //startTimer(); // can't get this to work together with findout dropdown filter 
```

Comment:
It could be helpful to console log when the start function is being called, as it is easier to spot errors in restart / play again